README.md

```diff
@@ -85,29 +85,12 @@ MyLA allows you to configure the primary color of the user interface (i.e., the
 
 Other colors from the user interface are currently fixed and included in version control, as they are intentional design choices. You can see where color values are set in `assets/src/defaultPalette.js`.
 
-### LTI v1.1.1 Configuration
-Only basic LTI launches are supported at the moment (automatic account creation and redirection to the correct course). New courses are not added nor are course view options modified.
-
-The relative LTI launch url is `/lti/auth/` (ex: `https://example.com/lti/auth/`). *NOTE* The trailing slash after `/auth/` is required! You'll get an error about CSRF if you have the wrong URL! :)
-
-#### Canvas Notes
-* When you configure the tool in Canvas the "Privacy" option must be changed from Anonymous to Public to pass along the user information required.
-
-* You should use the [XML builder](https://www.edu-apps.org/build_xml.html) to generate and paste the XML that rather than manually adding it. This will allow options like "Course Navigation" extension.
+### LTI v1.3 Configuration
+* MyLA Supports LTI 1.3 and using [pylti1.3](https://github.com/dmitry-viskov/pylti1.3) library for LTI validation 
```

Comment:
Is "validation" the right word?  Isn't it authentication, authentication, or handling?  Or maybe it doesn't need to be that specific, just remove the "library for LTI validation" part altogether.